toplevel/byte/topeval.ml

```diff
@@ -45,9 +45,15 @@ module EvalBase = struct
 
   let eval_ident id =
     if Ident.persistent id || Ident.global id then begin
```

Comment:
Note that you can remove the first test `Ident.persistent` which is redundant with `Ident.global`.